Packs/AzureWAF/README.md

```diff
@@ -79,11 +81,19 @@ To configure Microsoft Azure WAF to send logs to Cortex XSIAM, follow the below
 3. Select **+ Add diagnostic setting** to create a new integration or select **Edit setting** for an existing integration.
 4. Enter a **Diagnostic setting name**. If you're editing an existing integration, you can't change the name.
 5. Select the log categories that you want to stream. Refer to the **Log Normalization** section for the supported log categories for normalization.
-6. Select the **Stream to an event hub** checkbox.
-7. Select the Azure subscription, Event Hubs namespace, and optional event hub where you want to route the logs.
+6. Select streaming method and storing method
+
+   6.1. If you wish to use Event Hub and storage account to save the diagnostic logs select the **Stream to an event hub** checkbox.
+   
+   6.2. If you wish Log Analytics to store the logs select **Send To Log Analytics** checkbox and choose **Log Analytics Workspace**.
```

Comment:
6.2. Select the **Send To Log Analytics** checkbox and choose **Log Analytics Workspace** to use a Log Analytics workspace to store the logs.